xml/src/main/resources/ehcache-core.xsd

```diff
@@ -55,10 +55,26 @@
 
   <xs:complexType name="service-type">
     <xs:sequence>
-      <xs:any namespace="##other" minOccurs="1" maxOccurs="1"/>
+      <xs:element name="serializer-default" type="ehcache:serializer-type" minOccurs="0" maxOccurs="1"/>
```

Comment:
I _think_ this is wrong... doesn't this allow for both elements to be present? It should be either or, not both